### PR TITLE
Check message response codes also to avoid returning success on alrea…

### PR DESCRIPTION
**Pull request by UnitRoot22**
…dy captured messages

### CODE REVIEW

File: gateways/authorizenet/authorizenet.go [hunk 1]
```diff
@@ -65,7 +65,8 @@ func (client *AuthorizeNetClient) Capture(request *sleet.CaptureRequest) (*sleet
 		return nil, err
 	}
 
-	if authorizeNetResponse.TransactionResponse.ResponseCode != ResponseCodeApproved {
+	if authorizeNetResponse.TransactionResponse.ResponseCode != ResponseCodeApproved ||
+		isAlreadyCaptured(authorizeNetResponse.TransactionResponse) {
```
Comment (UnitRoot22) [inline gateways/authorizenet/authorizenet.go:69]: If condition becomes any more complex, will split it out into its own set of approved message response codes

File: gateways/authorizenet/types.go [hunk 2]
```diff
@@ -21,6 +21,10 @@ const (
 	ResponseCodeHeld     ResponseCode = "4"
 )
 
+const (
```
Comment (UnitRoot22) [inline gateways/authorizenet/types.go:24]: We can probably expand this as time goes on, but right now, this is to fix the one issue mentioned in this ticket 
https://app.asana.com/0/1201437212753355/1201868442243928

File: gateways/authorizenet/test_data/authAlreadyCapturedResponse.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  "messages": {
+    "resultCode": "Ok",
+    "message": [
+      {
+        "code": "I00001",
+        "text": "Successful."
+      }
+    ]
+  }
+}
```
Comment (yuweihsu01) [inline gateways/authorizenet/test_data/authAlreadyCapturedResponse.json:32]: nit: new line

**Review by yuweihsu01 — APPROVED** [review]
LGTM

**Pull request merged**